goforit.go

```diff
@@ -203,6 +212,9 @@ func (g *goforit) Enabled(ctx context.Context, name string, properties map[strin
 		}()
 	default:
 	}
+	if g.evalCB != nil {
+		defer func() { g.evalCB(name, enabled) }()
```

Comment:
Could you add a comment explaining why this isn't just `defer g.evalCB(name, enabled)`?

Reply:
Done!